openshift/openshift-copies.go

```diff
@@ -394,14 +399,20 @@ func validateAuthInfo(authInfoName string, authInfo clientcmdAuthInfo) []error {
 
 		if len(authInfo.ClientCertificate) != 0 {
 			clientCertFile, err := os.Open(authInfo.ClientCertificate)
-			defer clientCertFile.Close()
+			defer func() {
+				err = clientCertFile.Close()
+				logrus.Debugf("error closing client cert CA: %v", err)
```

Comment:
The two copy&pasted errors here and below are incorrect.

Reply:
Sorry, I'm not clear about these copy&pasted errors

Reply:
The text of the 3 `Debugf` messages is the same (“closing client CA”), but only one of them actually closes a client CA; the other two should be updated to match what actually failed (client certificate/key file).